packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react-native';
+import { PaddedView } from '@apollosproject/ui-kit';
+
+import PrayerCard from './PrayerCard';
+
+const cardData = {
+  imageSource: {
+    uri: 'https://fillmurray.com/400/600',
+  },
+  name: 'Bill',
+  prayer:
```

Comment:
This is `text` in the `PrayerCard`.
```suggestion
  text:
```